src/App.js

```diff
@@ -1,13 +1,58 @@
-import React from 'react';
-
+import React, { Component } from 'react';
+import { GoodsList } from './components/GoodsList';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends Component {
+  state = {
+    goods: [],
+  }
+
+  goodsFromServer = (value) => {
```

Comment:
```suggestion
  getGoodsFromServer = (value) => {
```
название функции - глагол